### DISCUSSION TITLE
Redesign /busy page

### DISCUSSION THREAD

**Discussion by fatkodima** [sidekiq/sidekiq#6524] [Ideas]
Currently, the /busy page have a few sections displayed in it - redis stats, active processes and running jobs. This can be hard to navigate when there are lots of active processes - need to scroll to the bottom of the page. Also the page can hang in the browser. And active processes are not much useful compared to running jobs. 

I don't have exact ideas how to improve it, except like maybe separating processes + redis stats from currently running jobs into separate pages.

This is just a suggestion, if not worth it - can close.

**Comment by raybaxter** [comment]
I agree. In my daily usage of Sidekiq, the Status section of the /busy page isn't useful and I am most often looking for Jobs, not processes

**Comment by mperham** [comment]
I know the processes and jobs tables can get huge for larger users. Should we provide a filter of some sort? How about a dropdown gadget to show/hide each table's content? Other ideas welcome.

**Reply by fatkodima** [reply]
A filter is useful when searching, but not much when just looking. Maybe add a new "Processes" page with all the processes info and leave only all the running jobs info in /busy? And put redis stats somewhere.

**Reply by mperham** [reply]
UX is not my strength, I'm not sure how to rejigger the UI in a way that solves the problem and scales from small to big screens. The easiest thing to do is render the Processes tab as usual but change its table contents CSS to default to `display:none` with a widget to expand the table. That way the Jobs table is the focus by default.

**Reply by agrieser-healthie** [reply]
Would be nice to have jobs on a separate page from processes. When I'm on `/busy`, 99% of the time I'm looking for a job, and we have so many processes that the jobs section of the page is _way_ down there. As a datapoint, this screen is `27871` pixels in height, or about 30 screen lengths on a macbook

Would also be great to have a filter like the other pages (retries etc). But if I had to pick one thing, it would be separating processes and jobs pages

**Reply by mperham** [reply]
Maybe two sub-tabs?
